acsylla/_cython/session/session.pyx

```diff
@@ -109,7 +109,7 @@ cdef class Session:
 
         return result
 
-    async def create_prepared(self, str statement, object timeout=None, object consistency=None):
+    async def create_prepared(self, str statement, object timeout=None, object consistency=None, object serial_consistency=None):
```

Comment:
what is `serial_consistency` doing at Cassandra/Scylla level? just asking

Reply:
`serial_consistency` used for lightweight transactions, by default the value is taken from` consistency` i think we can not use it in statement context but I try to implement maximum api https://docs.datastax.com/en/developer/cpp-driver /2.16/api/struct.CassStatement/